3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,92 @@
+
+//A future age calculator
+let yearOfBirth = 1993 ;
+
+let yearFuture = 2030 ;
+
+const age = yearFuture - yearOfBirth ;
```

Comment:
Great declaration of variables!
A little comment would be, always use `let` for variables that can change and `const` for values that stay the same.

In this case age could definitely be a `const`. `yearOfBirth` and `yearFuture`  could also be `const` unless you are planning to assign them different values later on.